packages/core/src/Popup.js

```diff
@@ -107,7 +134,15 @@ function Popup({
             <div className={BEM.container}>
                 <div className={BEM.body}>
                     {icon && wrapIfNotElement(icon, { with: PopupIcon, via: 'type' })}
-                    {message && wrapIfNotElement(message, { with: PopupMessage, via: 'text' })}
+                    {messageArea}
+                    {(!messageArea && message) && isValidElement(message)
```

Comment:
感覺補個括號，下面的三元運算子會比較看得懂條件是基於這一整條而不是只有 `isValidElement`。
```suggestion
                    {((!messageArea && message) && isValidElement(message))
```

Reply:
這樣意思不太一樣
原本的意思是 (!messageArea && message) 如果是 false 就不印任何東西
如果是 true 再判斷 isValidElement(message) 決定要印什麼
不過感謝你的提醒，我更新一下排版應該可以更好理解，避免誤會